dbus_service.py

```diff
@@ -546,6 +570,7 @@ def get_values_for_inverter(self):
             else:
                 pvyield = meter_data["inverters"][self.pvinverternumber]["AC"]["0"]["YieldTotal"]["v"]
             voltage = meter_data["inverters"][self.pvinverternumber]["AC"]["0"]["Voltage"]["v"]
+            dc_voltage = meter_data["inverters"][self.pvinverternumber]["DC"]["0"]["Voltage"]["v"]
```

Comment:
Please add some tests to test if values are extracted correctly (using the existing test data). Should not be complicated, we already have similar tests.

Reply:
Done